Dockerfile-ubuntu

```diff
@@ -0,0 +1,20 @@
+FROM golang:latest as builder
+
+RUN go get github.com/asobti/kube-monkey
+WORKDIR $GOPATH/src/github.com/asobti/kube-monkey
+RUN make build
```

Comment:
Able to succeed build on dockerhub 

https://hub.docker.com/r/nsidhaye/kube-monkey/builds/bifycs3upjrqgm6dkurrkl7/